src/noted-objects-analysis/members-parser.js

```diff
@@ -153,13 +153,13 @@ function setMemberDefinitions(definition, comment, modifiers, constructor = fals
 
     const types = getParams(comment);
     const typeArr = Object.keys(types);
-    const [, memberof] = /memberof (.*)/.exec(comment) || [];
+    const [, memberof] = /member[Oo]f (.*)/.exec(comment) || [];
 
     if (!isDeprecated && Object.keys(types).length !== args.length) {
       console.log(
         'TSgen: Parameter length of JSDocs (' + Object.keys(types).length + ') ' +
         'and definition arguments (' + args.length + ') ' +
-        'do not match for ' + memberof + ' ' + definition
+        'do not match for function ' + definition
```

Comment:
Not having available in error description name of `meberof` is likely to make it more difficult to identify where the issue occurred.

Reply:
definition would include the entire namespace.  So putting memberof here would be repetitive.